tracing/tracing.go

```diff
@@ -0,0 +1,122 @@
+package tracing
```

Comment:
This is a tracing module meant to be imported by an app that also imported a wag client. 

It allows for instrumenting tracing using our defaults.